docs/cli/deriva-sitemap-cli.md

```diff
@@ -0,0 +1,48 @@
+# deriva-sitemap-cli
```

Comment:
You'll want to edit `docs/index.rst` and add a reference to this file in the "Command Line Interfaces" toctree.